biometrics/minor_contamination.py

```diff
@@ -145,13 +149,14 @@ def plot(self, samples, outdir):
             mode='markers'
         ))
 
+        if len(plot_data) > 0:
+            fig.update_layout(yaxis = dict(range=(-0.005, plot_data['MAF'].max()*1.05)))
 
-        ticks = plot_data[['sample_name', 'index']].drop_duplicates()
+        ticks = data[['sample_name', 'index']].drop_duplicates()
```

Comment:
not sure exactly how this works, but the fact that `drop_duplicates()` is called after the data has been plotted makes me worry that maybe the original data has duplicates in it, not sure though

Reply:
this is just so I can get a unique set of samples (in the same order as it was plotted. I then use this data to update the x axis ticks. it's probably not the most efficient way of doing things, but it works